src/System.Runtime/tests/TestCollectibleAssembly/TestCollectibleAssembly.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace TestCollectibleAssembly
+{
+    public class MyTestClass
+    {
+        public bool MyField = false;
+        public int MyProperty => 1;
+        public int MyMethod() { return 1 * 1; }
```

Comment:
Could you please add `public T MyGenericMethod<T>(T arg1) { return arg1; }` to MyTestClass and also static methods and Types to both test classes?